src/plugins/github/fetchReposForOrg.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import {type RepoId, makeRepoId} from "../../core/repoId";
+import dedent from "../../util/dedent";
+
+const GITHUB_GRAPHQL_SERVER = "https://api.github.com/graphql";
+async function ghquery(query, token): any {
+  const body = JSON.stringify({query});
+  const fetchOptions = {
+    method: "POST",
+    body,
+    headers: {
+      Authorization: `bearer ${token}`,
+    },
+  };
+  const response = await fetch(GITHUB_GRAPHQL_SERVER, fetchOptions);
+  const json = await response.json();
+  if (json.errors) {
+    throw json.errors;
+  }
+  return json;
+}
+
+export async function fetchReposForOrg(
+  org: string,
+  token: string
+): Promise<$ReadOnlyArray<RepoId>> {
+  const query = dedent`\
+  {
+    search(query: "org:${org}", type: REPOSITORY, first: 50) {
```

Comment:
> ```js
> "org:${org}"
> ```

Please don’t do this. It’s an injection vector. There’s no need to open
this up.

Suppose, for instance, that you exposed a simple web service that
displayed SourceCred for a user-provided organization on request. It so
happens that my favorite GitHub organization is called

    ") { totalCount } } mutation Z { deleteProject(…) } query Q2 { search(query: "

which is unfortunate for whoever owns your API key.

That “this is draft/prototype code” is not a sufficient excuse. Draft
and prototype code turns into production code very quickly and without
anyone noticing. In the early days of SourceCred I made sure that even
our prototype code did not have this exact vulnerability.

We have a perfectly safe mechanism for doing this: use GraphQL variables
rather than string interpolation.